### DISCUSSION TITLE
Aggiornare da Debian 11 a Debian 12

### DISCUSSION THREAD

**Discussion by avvalex** [indomus/forum#3164] [GENERALE - Discussioni generiche]
Buongiorno ragazzi, domanda stupida: mi ritrovo un mini PC con Debian 11 (nemmeno molto aggiornato), con tutto ciò che mi serve in Docker.
Vorrei passare a Debian 12, prima di aggiornare anche Frigate e installare un Coral TPU, ma non avrei proprio voglia di rifare tutto da zero.
Secondo voi faccio una cavolata se provo ad aggiornare Debian 11, e poi aggiorno al 12, senza fare una installazione pulita da zero?

**Comment by KC-inDomus** [comment]
Sta per uscire debian 13, io aspetterei.

**Reply by avvalex** [reply]
Ma avrei la possibilità di fare un aggiornamento a caldo da 11 a 13? Oppure dovrei cmq rispettare la catena (aggiornare l'11, pregare, fare upgrade a 12, aggiornare il 12, pregare, fare upgrade a 13)?

**Reply by KC-inDomus** [reply]
puoi fare un po' tutto quello che hai elencato
